kuma/wiki/tasks.py

```diff
@@ -112,11 +112,7 @@ def build_json_data_for_document_task(pk, stale):
 
 @receiver(render_done)
 def build_json_data_handler(sender, instance, **kwargs):
-    try:
-        build_json_data_for_document_task.delay(instance.pk, stale=False)
-    except:
-        logging.error('JSON metadata build task failed',
-                      exc_info=True)
+    build_json_data_for_document_task.delay(instance.pk, stale=False)
```

Comment:
well, this was never safe but actually hid a potential scheduling error behind a blank except. if there is a reason why scheduling the task doesn't work we want to know via an exception handling, not a log entry